### PR TITLE
Accessibility add alt to images

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/fluxcd/website/issues/864

### CODE REVIEW

**Comment by Arhell** [comment]
cc @dholbach

File: layouts/shortcodes/blocks/resource.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 {{ $title := .Get "title" | default "Video title" }}
 
 <div class="col-lg-4 mb-3 mb-lg-0 text-center ">
-  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" src="{{ $thumbnail }}"></a>
+  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" alt="image" src="{{ $thumbnail }}"></a>
```
Comment (dholbach) [inline layouts/shortcodes/blocks/resource.html:13]: Maybe this could be the video title instead?

Comment (Arhell) [reply layouts/shortcodes/blocks/resource.html]: I agree, thanks

File: layouts/shortcodes/blocks/resource.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 {{ $title := .Get "title" | default "Video title" }}
 
 <div class="col-lg-4 mb-3 mb-lg-0 text-center ">
-  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" src="{{ $thumbnail }}"></a>
+  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" alt="video" src="{{ $thumbnail }}"></a>
```
Comment (dholbach) [inline layouts/shortcodes/blocks/resource.html:13]:
```suggestion
  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" alt="{{ $title | markdownify }}" src="{{ $thumbnail }}"></a>
```

how about this? can you try it and see if it works for you? maybe add a `title` attribute as well?

Comment (Arhell) [reply layouts/shortcodes/blocks/resource.html]: Done

**Review by dholbach — APPROVED** [review]
Thanks @Arhell for the PR! :ferris_wheel:

**Pull request merged**